README.md

```diff
@@ -17,4 +17,105 @@ From a cloned copy of this repository, sample data can be loaded into Movilo fro
 </div>
 
 #### Wireframe:
-<img src="https://drive.google.com/uc?export=view&id=17jPsTX7J_bDDp0DaTn-C0XWKU70Gkpin">
\ No newline at end of file
+<img src="https://drive.google.com/uc?export=view&id=17jPsTX7J_bDDp0DaTn-C0XWKU70Gkpin">
+
+# Movilo ![CI/CD](https://github.com/cs481-ekh/f22-viz-kids/workflows/CI/badge.svg)
+BSU CS481 Capstone project
+
+
+#### Deployments:
+* Official: https://sdp.boisestate.edu/movilo/ 
```

Comment:
This should be https://sdp.boisestate.edu/f22-movilo/

Dr. Henderson clarified today that his URL naming convention includes the semester